OpenRA.Game/Selection.cs

```diff
@@ -52,14 +64,26 @@ public void Combine(World world, IEnumerable<Actor> newSelection, bool isCombine
 					actors.UnionWith(newSelection);
 				else
 				{
+					if (stickyselect == true)
+					{
+						if (newSelection.Count() == 0)
+							actors.UnionWith(newSelection);
+						else
+						actors.Clear();
+						actors.UnionWith(newSelection);
```

Comment:
Here as well: one more level of indentation, and braces are missing on the else block